### PR TITLE
sql-join | Rich Trunzo 

### PR DESCRIPTION
**Pull request by richtrunzo**
Screen Shot 2021-01-12 at 1 17 33 PM

### CODE REVIEW

**Comment by richtrunzo** [comment]
Screen Shot 2021-01-12 at 1 23 44 PM
Screen Shot 2021-01-12 at 1 29 46 PM
Screen Shot 2021-01-12 at 2 42 34 PM
Screen Shot 2021-01-12 at 2 54 03 PM

File: sql-join/where-are-they.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+select "addresses"."line1",
+       "cities"."name",
+       "addresses"."district"
+from "addresses"
+join "cities" using ("cityId");
```
Comment (BrettAlbright) [inline sql-join/where-are-they.sql]: you are missing the country names good sir!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update needed

File: sql-join/where-are-they.sql (hunk 1 above)
Comment (richtrunzo) [reply sql-join/where-are-they.sql]: Screen Shot 2021-01-12 at 3 17 22 PM

Comment (richtrunzo) [reply sql-join/where-are-they.sql]: Updated 👍

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**